utam-compiler/src/main/java/utam/compiler/translator/ClassSerializer.java

```diff
@@ -95,6 +97,15 @@ public String toString() {
             .filter(method -> method.isPublic() || translationContext.getUsedPrivateMethods().contains(method.getDeclaration().getName()))
             .flatMap(method -> getMethodDeclaration(method).stream()).forEach(out::add);
     out.add(NEW_LINE);
+    source.getMethods().stream()
+        .filter(method -> (method.isPublic() || translationContext.getUsedPrivateMethods().contains(method.getDeclaration().getName())) && method.isElementMethod())
+        .map(method -> method.getDeclaration().getReturnType())
+        .map(returnType -> returnType instanceof TypeUtilities.ListOf ? ((TypeUtilities.ListOf)returnType).getElementType() : returnType)
+        .map(returnType -> String.format(
+            "public static class %s extends BasePageElement implements %s {}",
+            returnType.getSimpleName() + "Impl",
+            returnType.getSimpleName()))
+        .collect(Collectors.toList()).forEach(out::add);
```

Comment:
collecting is redundant here, just forEach

Reply:
Indeed it is. Good catch. Fixed in next commit.